pkg/tsdbctl/delete.go

```diff
@@ -51,6 +53,8 @@ func newDeleteCommandeer(rootCommandeer *RootCommandeer) *delCommandeer {
 
 	cmd.Flags().BoolVarP(&commandeer.delConfig, "del-config", "d", false, "Delete the TSDB config as well")
 	cmd.Flags().BoolVarP(&commandeer.force, "force", "f", false, "Delete all elements even if some steps fail")
+	cmd.Flags().Int64VarP(&commandeer.fromTime, "from-time", "r", 0, "Delete partitions from this time and on")
+	cmd.Flags().Int64VarP(&commandeer.toTime, "to-time", "t", 0, "Delete partitions up to this time")
 	commandeer.cmd = cmd
```

Comment:
@dinal better use the string representation of time in CLI (can work with multiple date semantics), see query commanf begin/end times